<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>New Mail Notification</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="3" colspan="5">
           <widget class="ColorButton" name="btnNotificationColor">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="5">
           <widget class="QSlider" name="sliderBlinkingSpeed">
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="pageStep">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Bold:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Blinking speed:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QSpinBox" name="notificationFontWeight">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This changes the font thickness, i.e. makes font bold.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Font style:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Default notification color:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="3">
           <widget class="QFontComboBox" name="notificationFont"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Icon (Ctrl-click to reset):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="5">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QToolButton" name="btnNotificationIcon">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="boxNotificationIconUnread">
              <property name="text">
               <string>Different icon when unread:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnNotificationIconUnread">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Fix the unread message count</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Use this tool if your unread counter is non-zero and there are no unread messages in your folders:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFixUnreadCount">
            <property name="text">
             <string>Fix</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAccounts">
      <attribute name="title">
       <string>Monitoring</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Method to parser unread notifications:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxParserSelection"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupThunderbirdProfilePath">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Please choose the Thunderbird profile database to monitor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="leProfilePath"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAccounts">
         <property name="title">
          <string>Accounts to monitor</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTreeView" name="treeAccounts">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="itemsExpandable">
               <bool>false</bool>
              </property>
              <property name="expandsOnDoubleClick">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnAccountAdd">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnAccountEdit">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnAccountRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Hiding</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Launch and Hiding</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="boxLaunchThunderbirdAtStart">
              <property name="text">
               <string>Start Thunderbird when Birdtray starts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="boxHideWindowAtStart">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>and hide it</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="boxStopThunderbirdOnExit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Terminate Thunderbird when quitting Birdtray </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="boxShowHideThunderbird">
            <property name="text">
             <string>Hide/show Thunderbird window when clicking on tray icon</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="boxHideWhenMinimized">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Hide Thunderbird window when it is minimized</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="boxMonitorThunderbirdWindow">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Show red X in tray icon if Thunderbird is not running</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="boxRestartThunderbird">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Restart Thunderbird if it was closed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="boxHideWindowAtRestart">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>and hide it</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>103</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>New Email</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="boxEnableNewEmail">
         <property name="text">
          <string>Enable New Email submenu</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTreeView" name="treeNewEmails">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="expandsOnDoubleClick">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewEmailAdd">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewEmailEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNewEmailDelete">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Please do not change those settings unless you understand whatt you're doing.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Path to Thunderbird executable:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leThunderbirdBinary"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Thunderbird window name pattern:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leThunderbirdWindowMatch"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Minimum notification font size:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinMinimumFontSize">
            <property name="suffix">
             <string> points</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>512</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QCheckBox" name="boxBlinkingUsesAlpha">
            <property name="text">
             <string>When blinking, use the fade-in/fade-out transition (increases CPU use!)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>179</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="browserAbout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>boxLaunchThunderbirdAtStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>boxHideWindowAtStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxRestartThunderbird</sender>
   <signal>toggled(bool)</signal>
   <receiver>boxHideWindowAtRestart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxEnableNewEmail</sender>
   <signal>toggled(bool)</signal>
   <receiver>treeNewEmails</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxEnableNewEmail</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnNewEmailAdd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxEnableNewEmail</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnNewEmailEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxEnableNewEmail</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnNewEmailDelete</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxNotificationIconUnread</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnNotificationIconUnread</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
